.github/workflows/router-ci.yaml

```diff
@@ -211,21 +212,9 @@ jobs:
             redis-cli -u "redis://cosmo:test@127.0.0.1:$port" ping
             echo "ACL user 'cosmo' created with full access on port $port"
           done
-      - name: Run Integration tests
-        if: matrix.test_target == 'non_flaky'
+      - name: Run Integration tests ${{ matrix.test_target }}
         working-directory: ./router-tests
-        run: make test test_params="-run '^Test[^(Flaky)]' --timeout=5m --parallel 10"
-      - name: Run Flaky Integration tests
-        if: matrix.test_target == 'flaky'
-        uses: nick-fields/retry@v3
-        with:
```

Comment:
I know some of the tests have internal retries (atleast from the logs), but what do you think about retrying using the .github action in addition to internal retries (if an internal retry in the tests exists).